### PR TITLE
Initial experiments for supporting verify flags.

### PR DESCRIPTION
**Pull request by oesolutions**
Hi Craig,

I'm looking into adding some support for the verify flags.  Does it seem like I'm on the right track or am I way off base?

Thanks, Chris

### CODE REVIEW

File: src/main/java/org/redline_rpm/Builder.java [hunk 1]
```diff
@@ -845,6 +845,28 @@ public void addFile( final String path, final File source, final int mode, final
...
+     *
+     * @param path the absolute path at which this file will be installed.
+     * @param source the file content to include in this rpm.
+     * @param mode the mode of the target file in standard three octet notation, or -1 for default.
+     * @param dirmode the mode of the parent directories in standard three octet notation, or -1 for default.
+     * @param directive directive indicating special handling for this file.
+     * @param uname user owner for the given file, or null for default user.
+     * @param gname group owner for the given file, or null for default group.
+     * @param addParents whether to create parent directories for the file, defaults to true for other methods.
+     * @param verifyFlags verify flags
```
Comment (oesolutions) [inline src/main/java/org/redline_rpm/Builder.java]: Right now I'm using int for passing around the verification flags, however, I'd add something similar to Directive.

**Comment by craigwblake** [comment]
Hi Chris, looks fine to me.  If you want to add something like Directive it should really be an enum - Directive was originally added back before Java 5 was out, hence the old style.  Maybe one day we'll get Directive updated to be an enum as well. ;)

I'm okay either pulling this in the interim or waiting for an update to add the enum, let me know which you prefer.

**Comment by oesolutions** [comment]
I've done zero testing so it's probably a good idea to wait. It was getting late when I did this and I just wanted to get your initial feedback before I put more effort into it. Thanks for taking a look.

**Comment by craigwblake** [comment]
Merging this as is since it's a simple addition and will allow some wider testing.

**Pull request merged**